app/views/groups/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= render partial: "shared/navigation", locals: {icon_type:'fa-solid', icon_name: 'arrow-left', title:"TRANSACTIONS", right:''} %>
+<div class="show-actions">
+  <%= link_to "Edit", edit_group_path(@group) %>
+  <%= button_to "Delete", @group, method: :delete %>
+</div>
+<%= render @group %>
```

Comment:
- [ ] On the `new` page to add a new category, please replace your Home button with a "Back" button (<) as stipulated in the project requirements so that when the user clicks on the "Back" button (<), the user navigates to the home page

Reply:
👍